### PR TITLE
Clean up leftovers about dex, api server proxy and permission controller

### PR DESCRIPTION
**Pull request by kwiatekus**
**Related issue(s)**
https://github.com/kyma-project/kyma/issues/11521

### CODE REVIEW

File: docs/05-technical-reference/06-custom-resources/custrsrc-kyma-installation.md [hunk 1]
```diff
@@ -34,8 +34,6 @@ spec:
     - name: "istio"
       namespace: "istio-system"
     - name: "provision-bundles"
-    - name: "dex"
-      namespace: "kyma-system"
     - name: "core"
```
Comment (strekm) [inline docs/05-technical-reference/06-custom-resources/custrsrc-kyma-installation.md:37]: core can also be removed from here

Comment (strekm) [reply docs/05-technical-reference/06-custom-resources/custrsrc-kyma-installation.md:37]: shouldn't this whole docu be removed? or at least replaced with describing components.yaml?

**Review by mmitoraj — COMMENTED** [review]
- Does the clean-up refer only to the `/docs` folder? I found also references to dex, IAM-kubeconfig, apiserver-proxy or permission controller in different README.md files in `/components` `/resources` and `/tests`. Is clean-up in those README.mds covered by another issue?
- I also found a reference to **apiserver-proxy** in `/docs/04-operation-guides/security/09-tls-certificates-security.md` line 63 and `/docs/04-operation-guides/troubleshooting/troubleshoot-certificates-gardener.md` line 10
- I also found a reference to **static user** in `/docs/04-operation-guides/security/03-02-authorization-in-kyma.md` lines 38 and 55
- **Permission Controller** appears not only in the removed by you section ### Namespace-wide authorization in `/docs/04-operation-guides/security/03-02-authorization-in-kyma.md` but also inthe ### Cluster-wide authorization section of the same doc (lines 22, and 27)

**Comment by strekm** [comment]
> * Does the clean-up refer only to the `/docs` folder? I found also references to dex, IAM-kubeconfig, apiserver-proxy or permission controller in different README.md files in `/components` `/resources` and `/tests`. Is clean-up in those README.mds covered by another issue?
@mmitoraj this will be covered in the removal of these components

File: docs/05-technical-reference/06-custom-resources/custrsrc-kyma-installation.md (hunk 1 above)
Comment (kwiatekus) [reply docs/05-technical-reference/06-custom-resources/custrsrc-kyma-installation.md:37]: Maybe.. yes. But please.. Not in this PR.
There should be probably dedicated PR covering new installation in 2.0

**Comment by kwiatekus** [comment]
Closing because this cleanup is laready covered in different PR

**Pull request closed without merging**